server/api/routes.md

```diff
@@ -0,0 +1,49 @@
+**USER: (/api/user)**
+
+* register guest user (POST) - occurs as soon as any user visits site (any page). Sets cookie (sessionId)
+* create user account (POST)
+* authenticate registered user (POST) - sets cookie (some version of userId)
```

Comment:
I think authentication should use the /auth routes and not overload the /api/user route
We should follow the pattern Eliot first introduced in the auth lecture  with a GET to a /auth/whoami route

Reply:
👍